src/functions-reference/index.qmd

```diff
@@ -14,8 +14,9 @@ format:
 ![](img/logo_tm.png "Stan Logo"){width=225 .column-margin .d-none .d-md-block}
 :::
 
-This is the reference for the functions defined in the Stan math
-library and available in the Stan programming language.
+This is the reference for the functions defined in the
+[Stan math library](https://github.com/stan-dev/math) and available in the
+[Stan](https://mc-stan.org/) programming language.
```

Comment:
maybe link to the language section of the Stan reference manual?  or stanc3 repo?   the problem with linking to mc-stan.org is that the website itself contains very little about Stan qua PPL.